recipe/meta.yaml

```diff
@@ -11,12 +11,12 @@ source:
   sha256: 012d716c8b9ed1e513fcc4b18e5af16a8791f51e6d1716baccf988ad355c5a1f
 
 build:
-  number: 3
+  number: 4
   script:
     - export OPAL_PREFIX=$PREFIX   # [mpi == "openmpi"]
+    - copy %RECIPE_DIR%\mpi-win64.cfg %SRC_DIR\mpi.cfg  # [win]
     - {{ PYTHON }} -m pip install --no-deps . -vv
```

Comment:
Just FYI, this way should also work:

```
set MPICFG=%RECIPE_DIR%\mpi-win64.cfg
```

Reply:
Ah, secret feature! It's probably better. btw does the section name has to be `[mpi]` to be picked up by the build script? Can it be a unique identifier if there's only one section in the cfg file?

Reply:
I think it should be possible:
https://github.com/mpi4py/mpi4py/blob/e486430cd1dc1bff5c6da2a88fa5c5c9f916d909/conf/mpidistutils.py#L335-L338

Reply:
I am messing around at the recipe level for now, because I noticed `MSMPI_LIB` is always appended with arch: https://github.com/mpi4py/mpi4py/blob/28eb557182bed368701165f99788bd2497d59f42/conf/mpiconfig.py#L208
but conda-forge's msmpi put the library in `LIBRARY_LIB` (without arch):
https://github.com/conda-forge/msmpi-feedstock/blob/08b93de852542324249ec1ea0b47c1b4b6aaec74/recipe/meta.yaml#L30
So without patching the build script itself we won't get the path right. Let's see if we can work it out...

Reply:
Oh, sorry, the environment variable name should be `MSMPI_LIB64`, and not `MSMPI_LIB`!

About the `MPICFG` secret feature, you can set `MPICFG=filename,secname` separated with a comma, the default for `secname` is `mpi`.

PS: I know this is a huge mess, but you have no idea how much time I've spent over the years to get this working.

Reply:
Ah....right, I parsed the code wrong...Not sure what I was thinking 🤪 But I think the `CCP_HOME` backdoor you left is precisely tailored for Conda-Forge's build system. Nothing is be better than a one-liner change in the recipe 😁